optuna/integration/allennlp.py

```diff
@@ -56,9 +55,13 @@ class AllenNLPExecutor(object):
     The interface may change without prior notice to correspond to the update.
 
     See the examples of `objective function <https://github.com/optuna/optuna/blob/
-    master/examples/allennlp/allennlp_jsonnet.py>`_ and
-    `config file <https://github.com/optuna/optuna/blob/master/
-    examples/allennlp/classifier.jsonnet>`_.
+    master/examples/allennlp/allennlp_jsonnet.py>`_.
+
+    .. note::
+        From Optuna v2.1.0, user has to cast their parameters by using methods in Jsonnet.
```

Comment:
A nit, but `user` is countable. How about using the plural form?
```suggestion
        From Optuna v2.1.0, users have to cast their parameters by using methods in Jsonnet.
```